src/commands/endpoints/access/add.ts

```diff
@@ -18,18 +18,19 @@ export default class EndpointsAccessAdd extends BaseCommand {
 
   static examples = [
     '$ heroku endpoints:access:add postgresql-rigid-37567 --account_ids 123456',
-    '$ heroku endpoints:access:add postgresql-rigid-37567 --account_ids 123456,78910',
+    '$ heroku endpoints:access:add postgresql-rigid-37567 --account_ids 123456 78910',
```

Comment:
The flag is `--account-ids` rather than `--account_ids`, no?

Reply:
And I think this would only work if invoked as 

```console
$ heroku endpoints:access:add postgresql-rigid-37567 --account-ids "123456 78910"
```

no? Which is kind of an awkward syntax. Is there another common delimiter that is not valid in ARNs that could work? Maybe `;` or `:`?